src/traces/box/index.js

```diff
@@ -10,10 +10,18 @@
 'use strict';
 
 var Plotly = require('../../plotly');
-var d3 = require('d3');
-var isNumeric = require('fast-isnumeric');
 
-var boxes = module.exports = {};
+var boxes = {
```

Comment:
Hmm. If it's pervasive in the code we may as well keep it the same. My main argument against it is to avoid [this](https://github.com/plotly/plotly.js/blob/master/src/traces/histogram/index.js#L27) and so that all the modules exports can be seen in one chunk, although it's not so much a concern with these files where the `exports` are oneliners.